docs/getting-started/delete-on-dockstore.rst

```diff
@@ -0,0 +1,46 @@
+Deleting an Entry on Dockstore
+===================================
+
+Tutorial Goals
+--------------
+
+-  Deleting an entry
+-  Ensuring that the entry does not re-appear on Dockstore
+
+This tutorial walks through the process of deleting entries from Dockstore.
+This action is permanent and after you delete the workflow, it will no longer appear on Dockstore.
+
+Requirements for Deletion
+-------------------------
+
+An entry can only be deleted if it satisfies all the following:
+
+-  Registered via GitHub Apps,
```

Comment:
Actually, I may have misread the conversation below. 
I think deletion in general works for all workflows (fulfilling the requirements below)

It's just the note on like 44 which applies only to GitHub apps and is probably fine to leave here (since we may eventually want a Bitbucket app, etc.)